sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -342,6 +493,84 @@ public async Task AnalyzeSentimentBatchConvenienceWithStatisticsTest()
             Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
         }
 
+        [Test]
+        public async Task AnalyzeSentimentBatchConvenienceWithStatisticsTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = new List<string>
+            {
+                "That was the best day of my life!. I had a lot of fun at the park.",
+                "I'm not sure how I feel about this product. It is complicated."
+            };
+
+            AnalyzeSentimentResultCollection results = await client.AnalyzeSentimentBatchAsync(documents, options: new AnalyzeSentimentOptions() { IncludeStatistics = true });
+
+            foreach (AnalyzeSentimentResult docs in results)
+            {
+                CheckAnalyzeSentimentProperties(docs.DocumentSentiment);
+            }
+
+            Assert.AreEqual("Positive", results[0].DocumentSentiment.Sentiment.ToString());
+            Assert.AreEqual("Negative", results[1].DocumentSentiment.Sentiment.ToString());
+
+            Assert.IsNotNull(results.Statistics.ValidDocumentCount);
+            Assert.IsNotNull(results.Statistics.DocumentCount);
+            Assert.IsNotNull(results.Statistics.TransactionCount);
+            Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
+        }
+
+        [Test]
+        public async Task AnalyzeSentimentBatchConvenienceWithStatisticsAndCancellationTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = new List<string>
+            {
+                "That was the best day of my life!. I had a lot of fun at the park.",
+                "I'm not sure how I feel about this product. It is complicated."
+            };
+
+            AnalyzeSentimentResultCollection results = await client.AnalyzeSentimentBatchAsync(documents, options: new AnalyzeSentimentOptions() { IncludeStatistics = true }, cancellationToken: default);
+
+            foreach (AnalyzeSentimentResult docs in results)
+            {
+                CheckAnalyzeSentimentProperties(docs.DocumentSentiment);
+            }
+
+            Assert.AreEqual("Positive", results[0].DocumentSentiment.Sentiment.ToString());
+            Assert.AreEqual("Negative", results[1].DocumentSentiment.Sentiment.ToString());
+
+            Assert.IsNotNull(results.Statistics.ValidDocumentCount);
+            Assert.IsNotNull(results.Statistics.DocumentCount);
+            Assert.IsNotNull(results.Statistics.TransactionCount);
+            Assert.IsNotNull(results.Statistics.InvalidDocumentCount);
+        }
+
+        [Test]
+        public async Task AnalyzeSentimentBatchConvenienceFullTest()
+        {
+            TextAnalyticsClient client = GetClient();
+            var documents = new List<string>
+            {
+                "That was the best day of my life!. I had a lot of fun at the park.",
+                "I'm not sure how I feel about this product. It is complicated."
+            };
```

Comment:
Is it normal to duplicate test inputs in your tests? I think it would be better to define those documents in a common place instead.

Reply:
Good point. Added to issue https://github.com/Azure/azure-sdk-for-net/issues/14619